hbase-server/src/main/java/org/apache/hadoop/hbase/wal/AbstractFSWALProvider.java

```diff
@@ -237,6 +237,9 @@ static void requestLogRoll(final WAL wal) {
   /** File Extension used while splitting an WAL into regions (HBASE-2312) */
   public static final String SPLITTING_EXT = "-splitting";
 
+  // Extension for the WAL where the split failed on one worker and is being retried on another.
+  public static final String RETRYING_EXT = ".retrying";
+
   /**
    * Pattern used to validate a WAL file name see {@link #validateWALFilename(String)} for
    * description.
```

Comment:
while splitting the wal for meta table. wal name can be rs.XXX.meta.retrying001. Do you think we should update the WAL_FILE_NAME_PATTERN. Althought in splitting we didn't check for valid wal name.